### DISCUSSION TITLE
Decision Explorer Linkages

### DISCUSSION THREAD

**Discussion by tulrichtrimble** [structurizr/ui#85] [Q&A]
Could someone explain ADR links in the Decision Explorer before I dig much further into the source? Decisions associated with nested contexts do not appear to be linked in the graph. Near as I can tell, decisions are only linked to a default root node. I'm sure I'm missing something and would like to understand as I advocate for C4 adoption across several sectors and dozens of teams.

The docs don't shed much light, but I'd be happy to update them once I have a handle on issue.

Cheers!

**Comment by simonbrowndotje** [comment]
The ADRs explorer is relatively simplistic ... it just shows links between ADRs in the same context. Example:

- Published ADR: https://structurizr.com/share/39459/decisions/adr-tools#5
- Source: https://raw.githubusercontent.com/structurizr/examples/main/java/src/main/java/com/structurizr/example/adrtools/decisions/0005-help-comments.md
- Graph:

<img width="1092" alt="Screenshot 2024-09-19 at 09 35 00" src="https://github.com/user-attachments/assets/368e257b-973b-4d76-8a3d-1f1185c40943">

**Reply by tulrichtrimble** [reply]
I see, Thank you. 
So it's primarily about ADR lineage. Are there any plans to develop a navigable ADR catalog linking the hierarchy? This  would be immensely useful for our need to consider designs, decisions, and resources across a dozen domains, each with dozens of systems, each containing dozens of containers, etc. Some of these decisions reference one another vertically, across contexts.

We're currently exploring how these disparate systems can reside among and be reused in workspaces.

**Reply by simonbrowndotje** [reply]
No plans. You might find that an EA tool better suits the level/scale you're trying to work at to be honest.
